src/common/models/split-combine/split-combine.mocha.ts

```diff
@@ -1,12 +1,20 @@
 'use strict';
 
 import { expect } from 'chai';
+import { testImmutableClass } from 'immutable-class/build/tester';
 
 import { $, Expression } from 'plywood';
 import { SplitCombine } from './split-combine';
 
 describe('SplitCombine', () => {
-  it('works in empty case', () => {
-
-  });
+    it('is an immutable class', () => {
+        testImmutableClass(SplitCombine, [
+            {
+                expression: {op: 'ref', name: 'language'}
+            },
+            {
+                expression: {op: 'ref', name: 'lookup'}
+            }
+        ]);
+    });
```

Comment:
indent using 2 spaces here please. Also, I know that SplitCombine can have more diverse things in it. It would be great to see a test case here that includes `bucketAction`, `sortAction`, and `limitAction`